app/serializers/review_serializer.rb

```diff
@@ -0,0 +1,3 @@
+class ReviewSerializer < ActiveModel::Serializer
+  attributes :id, :description, :votes
```

Comment:
remove all vote mentions until you have that feature built out. The reason I say this is because you will likely be managing votes differently